### ISSUE TITLE
loading  CamemBert  from torch.hub

### ISSUE THREAD

**Issue by Myname10** [pytorch/fairseq#1829]
Hey,I'm facing a problem concering loading CamemBert from torch.hub.

This what I did :
- pip install torch so the installed version is 1.4.0
- run these two lines of code on my jupyter notebook:
import torch
camembert = torch.hub.load('pytorch/fairseq', 'camembert.v0')
camembert.eval()  # disable dropout (or leave in train mode to finetune)

And got this error: 

![Capture d’écran de 2020-03-12 15-20-18](https://user-images.githubusercontent.com/44066130/76531138-0d945e80-6475-11ea-9af5-7cff8e63bf23.png)

**Comment by myleott** [comment]
Ah, due to a [recent change](https://github.com/pytorch/fairseq/commit/937535dba036dc3759a5334ab5b8110febbe8e6e), you'll need load a fresh copy with:
```
torch.hub.load(..., force_reload=True)
```
